File: be/src/main/java/nl/zzpmatcher/ZzpMatcherApplication.java
package nl.zzpmatcher;

import nl.zzpmatcher.business.UserRepository;
import org.springframework.beans.factory.annotation.Autowired;
import org.springframework.boot.CommandLineRunner;
import org.springframework.boot.SpringApplication;
import org.springframework.boot.autoconfigure.SpringBootApplication;

@SpringBootApplication
public class ZzpMatcherApplication implements CommandLineRunner{

	private final UserRepository userRepository;

	@Autowired
	public ZzpMatcherApplication(UserRepository userRepository) {
		this.userRepository = userRepository;
	}

	public static void main(String[] args) {
		SpringApplication.run(ZzpMatcherApplication.class, args);
	}

	@Override
	public void run(String... strings) throws Exception {
	}
}

File: be/src/main/java/nl/zzpmatcher/ZzpMatcherApplication.kt
package nl.zzpmatcher

import nl.zzpmatcher.userlogon.business.UserRepository
import org.springframework.beans.factory.annotation.Autowired
import org.springframework.boot.CommandLineRunner
import org.springframework.boot.SpringApplication
import org.springframework.boot.autoconfigure.SpringBootApplication

@SpringBootApplication
class ZzpMatcherApplication @Autowired
constructor(private val userRepository: UserRepository) : CommandLineRunner {

    @Throws(Exception::class)
    override fun run(vararg strings: String) {
    }

    companion object {

        @JvmStatic
        fun main(args: Array<String>) {
            SpringApplication.run(ZzpMatcherApplication::class.java, *args)
        }
    }
}
